File: front/src/app/models/stock/stock.ts


export class Stock {

  public id             ?: number;
  public code           ?: string;
  public name    ?: string;
  public wholesalePrice   ?: string;
  public retailPrice    ?: string;
  public itemType   ?: number;
  public inStock             ?: number;




    constructor(code: string, name: string, itemType: number) {

    this.code = code;
    this.name = name;
    this.itemType = itemType;

  }

}

File: back/SalseMart/Models/Stock.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SalseMart.Models
{
    public class Stock
    {
        public int id { get; set; }
        public string code { get; set; }
        public string name { get; set; }
        public decimal wholesalePrice { get; set; }
        public decimal retailPrice { get; set; }
        public int itemType { get; set; }
        public int inStock { get; set; }
    }
}